aioshelly/ble/const.py

```diff
@@ -19,22 +19,68 @@
 DEFAULT_DURATION_MS = -1
 
 BLE_CODE = """
-// aioshelly BLE script 1.0
-BLE.Scanner.Subscribe(function (ev, res) {
-    if (ev === BLE.Scanner.SCAN_RESULT) {
-        Shelly.emitEvent("%event_type%", [
-            %version%,
-            res.addr,
-            res.rssi,
-            btoa(res.advData),
-            btoa(res.scanRsp)
-        ]);
+// aioshelly BLE script 2.0
+const queueServeTimer = 450; // in ms, timer for events emitting
```

Comment:
Can we shorten this interval or try to think of a logic which will not require a big delay?
One of the advantages I liked with the Shelly BLE + Proxy is that the bluetooth response was very fast, with this change there is a noticeable delay with the events.
I use Shelly with door sensor to switch lights and buttons to switch monitors on a PC and the delay is noticeable